events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
+ * [ESP Library](https://github.com/earthspecies/library)
+
+
+## Participate: 
+
+The Earth Species Library is a collection of universally accessible bioacoustics datasets, ranging from humpback whales to zebra finches, tuned for machine learning. These datasets will enable machine learning researchers to train and validate novel machine learning methodologies and develop increasingly complex approaches to answer some of biology's most elusive questions. If you are interested in contributing your bioacoustic data and annotations to the Earth Species Library, please get in touch with [katie@earthspecies.org](mailto:katie@earthspecies.org). 
+
+## Getting Started: 
+
+*   [Introduction to using Collab](https://colab.research.google.com/notebooks/intro.ipynb)
+*   [Digital Signal Processing with Python](https://github.com/earthspecies/intro-to-DSP-with-python)
+
+## Resources: 
+
+*   [The “Cocktail Party Problem”: What Is It? How Can It Be Solved? And Why Should Animal Behaviorists Study It?](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2692487/)
+*   [Asteroid: The PyTorch-Based Audio Source Separation Toolkit for Researchers](https://arxiv.org/abs/2005.04132)
+*   [Wavesplit: End-to-End Speech Separation by Speaker Clustering](https://arxiv.org/abs/2002.08933)
+*   [Dual-Path Transformer Network: Direct Context-Aware Modeling for End-to-End Monaural Speech Separation](https://arxiv.org/abs/2007.13975)
+*   [Voice Separation with an Unknown Number of Multiple Speakers](https://arxiv.org/abs/2003.01531)
+*   [Separation of Overlapping Sources in Bioacoustic Mixtures](https://asa.scitation.org/doi/10.1121/10.0000932)
+*   [Separating Overlapping Bat Calls with a Bi-Directional Long Short-Term Memory Network](https://www.biorxiv.org/content/10.1101/2019.12.15.876656v1)
+*   [Music Source Separation in the Waveform Domain](https://arxiv.org/abs/1911.13254)
+*   [Deep Audio Prior](https://arxiv.org/abs/1912.10292)
+*   [Deep Learning for Monaural Speech Separation](https://paris.cs.illinois.edu/pubs/huang-icassp2014.pdf)
+*   [The Separation of Overlapped Dolphin Signature Whistle Based on Blind Source Separation](https://ieeexplore.ieee.org/document/8242534)
+*   [A Comparative Study of Blind Source Separation for Bioacoustics Sounds based on FastICA, PCA and NMF](https://www.sciencedirect.com/science/article/pii/S1877050918312468?via%3Dihub)
+*   [Investigating Deep Neural Transformations for Spectrogram-Based Musical Source Separation](https://arxiv.org/abs/1912.02591)
+*   [A Physiologically Inspired Model for Solving the Cocktail Party Problem](https://link.springer.com/content/pdf/10.1007/s10162-019-00732-4.pdf)
+*   [Audio Source Separation with Discriminative Scattering Networks](https://arxiv.org/abs/1412.7022)
+*   [Blind Nonnegative Source Separation Using Biological Neural Networks](https://arxiv.org/abs/1706.00382)
 
-## Resources
+# 
+**Miscellaneous Key Resources**
```

Comment:
What's the difference between these and the things in getting started? Can we combine?

Reply:
We could probably remove the Misc. Key Resources title - the resources above were kind of ordered in terms of relevance, and then as I proceeded through the literature, I randomly added more to the Misc. Key Resources section to focus on a smaller fraction of the 400-500 total

Reply:
Sounds good @pcbermant , I'll remove the title.